### PR TITLE
Minor Spelling Change in Contribute page and EinsteinPy Website Update

### PR DESCRIPTION
**Pull request by shreyasbapat**
A minor spelling fix

### CODE REVIEW

**Comment by hamogu** [comment]
Thanks! I can't see the circle CI failure because I don't have permissions, but I bet it's unrelated, I've had that with one of my PRs recently, too.

**Comment by shreyasbapat** [comment]
@hamogu I can fix the CI 😬
Will do it in a while.

**Comment by shreyasbapat** [comment]
Fixed!

File: affiliated/registry.json [hunk 1]
```diff
@@ -894,7 +894,7 @@
                 "testing": "Good",
                 "devstatus": "Heavy development",
                 "python3": "Yes",
-                "last-updated": "2019-07-23"
+                "last-updated": "2020-06-19"
```
Comment (bsipocz) [inline affiliated/registry.json:897]: this should not have been changed, I think this date is for the review status and relevant to see when the package needs a rereview, etc.

Comment (pllim) [reply affiliated/registry.json:897]: Ops, my bad, sorry! I reverted it in c393d44

**Comment by hamogu** [comment]
And it's funny that I just fixed the website independently in https://github.com/astropy/astropy.github.com/commit/527876153f18b6f0df8af34f5694e690c726a056 about an hour ago because CI did not pass otherwise and I did not know about this PR! Yeah for git merging both just fine.

**Comment by pllim** [comment]
Phew. Sorry!

**Pull request merged**